src/cast_core/src/actors/RunCommand.cpp

```diff
@@ -139,8 +141,9 @@ class Operation {
 
 private:
     void _run() {
-        bsoncxx::builder::stream::document document{};
-        auto view = _doc->view(document);
+        auto command = std::get<bsoncxx::document::view_or_value>(_commandExpr->evaluate(_rng));
```

Comment:
can there be a simpler way here? E.g. `_commandExpr->value(_rng)` that does L144-145 under the covers and returns a `bsoncxx::document::value`?

Reply:
Changing to encapsulate the `std::variant` made it so you can write `_commandExpr->value(_rng).getDocument()`. Is that simple enough? The only reason I did `auto view = command.view()` on the immediately following line was so I didn't have change the places where the `view` variable was being used already.

Regarding `bsoncxx::document::value`, I'd like to emphasize that we'd only ever return a `bsoncxx::document::view` from `Expression::evaluate()` if the underlying buffer was owned by the `UniqueExpression`. Using `.view()` ensures we aren't making unnecessary copies of the BSON document.